backend/python/app/graphql/mutations/auth_mutation.py

```diff
@@ -161,7 +161,7 @@ def mutate(root, info, first_name, last_name, email, password):
                     last_name=last_name,
                     email=email,
                     role="User",
-                    password=password
+                    password=password,
```

Comment:
Heads-up that https://github.com/uwblueprint/planet-read/pull/48 makes similar linting changes to you. (I think they're all the same and so merging should then go smoothly? ? ? don't remember)

Reply:
should require pretty trivial rebase on either #48 or #50